modules/ROOT/pages/migration/upgrading_4.0.0_5.0.0.adoc

```diff
@@ -0,0 +1,87 @@
+= Upgrading from 3.0.0 to 4.0.0
+:toc: right
+:description: This document describes the necessary steps when upgrading Infinite Scale from release 4.0.0 to 5.0.0.
+
+== Introduction
+
+{description}
+
+IMPORTANT: Read the important notes in the xref:migration/upgrading-ocis.adoc#introduction[Upgrading Infinite Scale] documentation before you start.
+ 
+== Upgrade Steps
+
+. Shut down the Infinite Scale instance.
+. Update Infinite Scale via:
++
+--
+* Docker
++
+[source,bash]
+----
+docker pull owncloud/ocis:5.0.0
+----
+
+* or get the binary from {ocis-downloadpage-url}?sort=time&order=desc[download.owncloud.com,window=_blank].
+--
+
+== Manage Breaking Changes
+
+* All environment variables that were marked for deprecation in Infinite Scale release 4.0.0 have finally been removed.
+* The default service registry has been changed.
+
+=== How to Identify if You Are Affected
```

Comment:
Is this capitalization style really what we want? Reminds me of:

<img width="461" alt="Screenshot 2024-03-01 at 08 52 04" src="https://github.com/owncloud/docs-ocis/assets/3532843/bdf1c15f-909b-4a43-ade3-ca6ac1ec33bc">

Reply:
Yes, it is used consistently in ALL docs and we keep doing that.

Reply:
I know. But what's the rule that kept the `to` and `if` lower case but everything else capitalized?

Reply:
We use [headlinecapitalisation.com](https://headlinecapitalization.com)
You can just c/p content and get the result to use.
If something does not match, we need to fix it.
Except proper names, they are written as the owner has defined them.
Hope this helps 😄

Reply:
lol ok.